ovs/extensions/healthcheck/helpers/cache.py

```diff
@@ -21,54 +21,61 @@
 class CacheHelper(object):
 
     client = PersistentFactory.get_client()
-    prefix = 'ovs-health-check'
+    prefix = 'health-check_'
 
     @staticmethod
-    def set(info):
+    def set(item, key=None):
         """
         Store the information to the config management
-        :param info:
+        :param item:
         :return:
         """
+        key = CacheHelper._generate_key(key=key)
+        return CacheHelper.client.set(key=key, value=item)
 
-        if isinstance(info, type(dict)):
-            info = CacheHelper._parse_data(info)
-        CacheHelper.client.set(CacheHelper.prefix, info)
+    @staticmethod
+    def append(item, key=None):
+        """
+        Appends the information to the value
+        Supports dicts, lists and sets
+        :param item:
+        :return:
+        """
+        supported_types = [dict, list]
+        retrieved_value = CacheHelper.get(key=key)
+        for item_type in supported_types:
+            if isinstance(item, item_type) and isinstance(retrieved_value, item_type):
+                if item_type == list:
+                    return CacheHelper.set(retrieved_value + item)
```

Comment:
Shouldn't you also pass the ```key``` here in case it has been passed into this function?